test/lib/headers_test.rb

```diff
@@ -139,6 +139,30 @@ def test_pathname_with_trailing_slash_if_present_with_query_lang_when_trailing_s
       assert_equal('/test/', headers.pathname_with_trailing_slash_if_present)
     end
 
+    def test_pathname_for_unencoded_url
+      Store.instance.update_settings('url_pattern' => 'query', 'lang_param_name' => 'lang')
```

Comment:
Why `'lang_param_name' => 'lang'` is needed?

Reply:
`Store.instance.update_settings` was not needed, thanks!!